scripts/run-local-ci.sh

```
@@ -316,19 +316,26 @@ run_benchmark() {
+        --cov=file_organizer \
+        --cov-branch \
+        --cov-report=term-missing \
+        --cov-report=xml \
+        --timeout=60 \
+        --override-ini=addopts= \
+        | tee "$report_path"
+      "$PYTHON_BIN" scripts/check_module_coverage_floor.py \
+        --report-path "$report_path" \
+        --baseline-path scripts/coverage/integration_module_floor_baseline.json
+      coverage report --fail-under=71.9
+    '
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)